docs/docs/configuration.md

```diff
@@ -358,8 +358,7 @@ or
 
 mbt sbom-gen --sbom-file-path sbom-path/test.sbom.xml
 ```
-
-Or, for `java or nodejs` type module, because their default builder is `maven or npm`, do not need to set `build-parameters.builder` attribute : 
+For the `java` or `nodejs` module types, since their default builder is `maven` or `npm`, do not need to set the `build-parameters.builder` attribute:
```

Comment:
For the `java` or `nodejs` module types, since their default builder is `maven` or `npm`, you do not need to set the `build-parameters.builder` attribute: